db/seeds.rb

```diff
@@ -5,10 +5,5 @@
 #
 # Please ensure that all templates are submitted to community-templates, then they will be synced in.
 
-# define all helpers here
-def format_errors(model = nil)
-  SeedHelper.format_errors(model)
```

Comment:
To get this in quickly, should we have a deprecation warning for next version and keep it until the next version?